fe/fe-core/src/test/java/com/starrocks/sql/plan/PlanFragmentTest.java

```diff
@@ -4032,47 +4032,6 @@ public void testMergeAggregateFailed() throws Exception {
                 "  0:OlapScanNode"));
     }
 
-    // todo(ywb) disable replicate join temporarily
-    public void testReplicatedJoin() throws Exception {
```

Comment:
why directory remove  this test?

Reply:
We need support replicate join later？

Reply:
maybe.  I think we could keep the test code.